packages/common/src/core/types/Embeds.ts

```diff
@@ -64,6 +64,12 @@ export interface IHubEmbedSurvey {
   kind: EmbedKind.feedback;
   /** survey123 id */
   id: string;
+  /** whether to show the survey header */
```

Comment:
I know this is how the survey card works today, but is it actually in the AC to expose these settings right now?

Reply:
No, it's not -- I won't expose them from the od-ui side, but thought I'd put it in the interface. I can take it out for this PR if you think that'd be better -- we can also expose them if we'd like.